packages/notte-agent/src/notte_agent/falco/agent.py

```
@@ -21,7 +21,7 @@
 from notte_core.credentials.base import BaseVault, LocatorAttributes
 from notte_core.llms.engine import LLMEngine
 from notte_core.utils.webp_replay import ScreenshotReplay, WebpReplay
-from notte_sdk.types import AgentCreateRequestDict
+from notte_sdk.types import AgentCreateRequestDict, AgentRunRequestDict
```

Ruff (0.11.9)

`notte_sdk.types.AgentRunRequestDict` imported but unused

Remove unused import: `notte_sdk.types.AgentRunRequestDict`

(F401)